src/main/kotlin/io/sip3/salto/ce/media/MediaManager.kt

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2018-2021 SIP3.IO, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.sip3.salto.ce.media
+
+import io.sip3.commons.domain.media.MediaControl
+import io.sip3.commons.domain.media.Recording
+import io.sip3.commons.domain.media.SdpSession
+import io.sip3.commons.vertx.annotations.Instance
+import io.sip3.commons.vertx.util.localPublish
+import io.sip3.commons.vertx.util.localRequest
+import io.sip3.salto.ce.RoutesCE
+import io.sip3.salto.ce.sip.SipTransaction
+import io.vertx.core.AbstractVerticle
+import mu.KotlinLogging
+
+/**
+ * Manages media feature
+ */
+@Instance
+open class MediaManager : AbstractVerticle() {
+
+    private val logger = KotlinLogging.logger {}
+
+    private var recordingEnabled: Boolean = false
+
+    override fun start() {
```

Comment:
`private var instances: Int = 1` accordingly to other verticles, let's define this parameter first